kubebuilder.mk

```diff
@@ -17,6 +17,10 @@ IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
+DOCKER ?= docker
+# Image to build protobugs
+PROTO_IMG ?= k8s.gcr.io/kube-cross:v1.13.5-1
```

Comment:
It looks like there's a [slightly newer version of this image now](https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/kube-cross?gcrImageListsize=30&pli=1), any reason not to use it?

Reply:
It didn't exist at the time, I'll update it